sdk/trace/batch_span_processor.go

```diff
@@ -121,6 +129,34 @@ func NewBatchSpanProcessor(exporter SpanExporter, options ...BatchSpanProcessorO
 	return bsp
 }
 
+func (bsp *batchSpanProcessor) configureSelfObservability() {
+	mp := otel.GetMeterProvider()
+	if !x.SelfObservability.Enabled() {
+		mp = metric.MeterProvider(noop.NewMeterProvider())
+	}
+	meter := mp.Meter(
+		selfObsScopeName,
+		metric.WithInstrumentationVersion(version()),
+	)
+
+	queueSizeCounter, err := meter.Int64ObservableUpDownCounter("otel.sdk.batch_span_processor.queue_size",
+		metric.WithUnit("{span}"),
+		metric.WithDescription("The number of ended spans currently enqueued by the processor."),
+	)
+	if err != nil {
+		otel.Handle(err)
```

Comment:
Should this return after the handle?

Reply:
The instrument returned by our SDK is still useable.  I was hoping to ensure the callback registration is never nil, so shutdown is always safe to call without checking if it is nil.